java/api/src/main/java/io/ray/api/utils/parallelactor/ParallelActorExecutor.java

```diff
@@ -0,0 +1,18 @@
+package io.ray.api.utils.parallelactor;
+
+import io.ray.api.function.RayFuncR;
+
+import java.util.List;
+
+public class ParallelActorExecutor {
```

Comment:
Is it OK to put it into the runtime module?

Reply:
If we hide the implementation of `ParallelActor` to runtime module, `ParallelActorExecutor` could be moved into runtime module as well.